src/database/schema/UserSchema.ts

```diff
@@ -316,7 +355,11 @@ class UserSchema implements BaseDbSchema {
       profilePicture: object?.profile_pic_path,
       updatedAt: object?.updated_at,
       username: object?.username,
-      isBanned: object?.is_banned
+      isBanned: object?.is_banned,
+      anon_user_info_color_code: object?.anon_user_info_color_code,
+      anon_user_info_color_name: object?.anon_user_info_color_name,
+      anon_user_info_emoji_code: object?.anon_user_info_emoji_code,
+      anon_user_info_emoji_name: object?.anon_user_info_emoji_name
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding new fields to the object returned by `fromDatabaseObject` should be fine as long as these fields are used correctly elsewhere in the code.
```diff
       username: object?.username,
-      isBanned: object?.is_banned
+      isBanned: object?.is_banned,
+      anon_user_info_color_code: object?.anon_user_info_color_code,
+      anon_user_info_color_name: object?.anon_user_info_color_name,
+      anon_user_info_emoji_code: object?.anon_user_info_emoji_code,
+      anon_user_info_emoji_name: object?.anon_user_info_emoji_name
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->